src/cmd/filter.go

```diff
@@ -3,24 +3,41 @@ package cmd
 import (
 	"encoding/json"
 	"fmt"
+
+	"github.com/creasty/defaults"
 	"github.com/opslevel/opslevel-go/v2023"
 
 	"github.com/opslevel/cli/common"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 )
 
 var createFilterCmd = &cobra.Command{
-	Use:        "filter NAME",
-	Short:      "Create a filter",
-	Long:       `Create a filter`,
-	Args:       cobra.ExactArgs(1),
-	ArgAliases: []string{"NAME"},
+	Use:   "filter",
+	Short: "Create a filter",
+	Long: `Create a filter
+
+cat << EOF | opslevel create filter -f -
+name: "Tier 1 Rails apps using RDS"
+connective: "and"
+predicates:
+  - key: "tier_index"
+    type: "equals"
+    value: "1"
+  - key: "filter_id"
+    type: "matches"
+    value: "Z2lkOi8vb3BzbGV2ZWwvRmlsdGVyLzEyNjQ"
+  - key: "tags"
+    keyData: "db"
+    type: "equals"
+    value: "rds"
+EOF`,
```

Comment:
```suggestion
	Long: `Create a filter`
    Example: `
cat << EOF | opslevel create filter -f -
name: "Tier 1 Rails apps using RDS"
connective: "and"
predicates:
  - key: "tier_index"
    type: "equals"
    value: "1"
  - key: "filter_id"
    type: "matches"
    value: "Z2lkOi8vb3BzbGV2ZWwvRmlsdGVyLzEyNjQ"
  - key: "tags"
    keyData: "db"
    type: "equals"
    value: "rds"
EOF`,
```

Reply:
the CLI framework has an "example" field where usage example are put.

Reply:
Added these suggestions